src/intrinsics/ecma262/Error.js

```diff
@@ -30,6 +30,13 @@ export function describeLocation(
   let locString = "";
   let displayName = "";
 
+  if (callerFn) {
```

Comment:
We already have an `if (callerFn)` condition below, maybe unify them?

Reply:
I do the checks twice as the logic in the middle of this function can cause it to never reach the end of this function.

Reply:
It would be nice to call it out explicitly or have a test that reproes it. Or somebody will break it.